lib/dotcom_web/live/system_status.ex

```diff
@@ -1,436 +1,32 @@
 defmodule DotcomWeb.Live.SystemStatus do
   @moduledoc """
-  A temporary LiveView for showing off the system status widget until we
-  put it into the homepage (and elsewhere).
+  A temporary LiveView showing PubSub and Nebulex for real-time cached data.
   """
 
   use DotcomWeb, :live_view
 
-  import Dotcom.Routes, only: [subway_route_ids: 0]
-  import DotcomWeb.Components.PlannedDisruptions
-  import DotcomWeb.Components.RouteSymbols
-  import DotcomWeb.Components.SystemStatus.StatusLabel
-  import DotcomWeb.Components.SystemStatus.SubwayStatus
+  import Dotcom.SystemStatus.CommuterRail, only: [commuter_rail_status: 0]
 
-  alias Dotcom.Alerts.Subway.Disruptions
-  alias Dotcom.SystemStatus
+  import DotcomWeb.Components.SystemStatus.CommuterRailStatus,
+    only: [alerts_commuter_rail_status: 1]
 
-  @date_time_module Application.compile_env!(:dotcom, :date_time_module)
+  def mount(_params, _session, socket) do
+    if connected?(socket) do
+      Phoenix.PubSub.subscribe(Dotcom.PubSub, "commuter-rail-system-status")
```

Comment:
Nice. How about using a naming scheme like `"system_status:commuter_rail"`, which gives us a clear template for adding to later (e.g. `"system_status:subway"` or `"system_status:Red"`)